iceoryx_posh/cmake/install_help_and_config.cmake

```diff
@@ -13,12 +13,18 @@
 # limitations under the License.
 #
 
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/../doc/website/FAQ.md
-	DESTINATION share/doc/iceoryx
-	COMPONENT dev)
+install(
+  FILES ${CMAKE_CURRENT_SOURCE_DIR}/../doc/website/FAQ.md
+  		${CMAKE_CURRENT_SOURCE_DIR}/../doc/website/getting-started/overview.md
```

Comment:
you could also add the new `installation-guide-for-contributors.md` from #481 here